build.gradle

```diff
@@ -51,10 +51,10 @@ dependencies {
   compile 'org.ow2.asm:asm:5.0.4'
   compile 'org.ow2.asm:asm-commons:5.0.4'
   compile 'org.ow2.asm:asm-tree:5.0.4'
-  compile 'org.elasticsearch:elasticsearch:7.2.0'
+  compile 'org.elasticsearch:elasticsearch:7.3.0'
```

Comment:
we need to bump org.apache.lucene:lucene-expressions:8.0.0 to 8.1.0 (c.f. https://github.com/elastic/elasticsearch/blob/v7.3.0/buildSrc/version.properties#L2)

Reply:
Done!